### ISSUE TITLE
Better positioning of camera output circle

### ISSUE THREAD

**Issue by ceskyDJ** [alyssaxuu/screenity#201]
Hi! I think the recording toolbox is positioned and animated just perfect! The camera output isn't that good though. I think, it should have a similar positioning system under the hood, so it will sense more reactive and "playable" (I'm sorry for my bad English, I hope it's clear what I mean). At least it should have the same gap between screen end and the circle as the toolbox has. Maybe there should be a minimal gap between the toolbox and the circle, too (so it will be always divided and the user could experience a easier manipulation process without overlapping the toolbox with the circle).

There are some screenshots with some not too nice situations (at least from my point of view):
![image](https://github.com/alyssaxuu/screenity/assets/48548230/cc8ea7d7-6cae-45f1-9252-86523190d11f)
![image](https://github.com/alyssaxuu/screenity/assets/48548230/f1e02b45-ff36-421c-82d7-f809f167a48e)
![image](https://github.com/alyssaxuu/screenity/assets/48548230/d1067e64-68ae-4565-a252-ae2fe0bbdcef)

**Comment by alyssaxuu** [comment]
I could reuse the same system the toolbar has definitely, I think I just kept it simple as to not make it too "flashy" since people often move the camera a lot while recording. I can try to improve those cases you mentioned too :)

**Comment by ceskyDJ** [comment]
You're right, it should be well-balanced. I don't know how it'll look like/behave, when you add some of the noted improvements, so I just recommended more things and the choice what to implement is all yours. I believe you choose well. The overall user experience of your extension is incredibly awesome, so I have no doubt it will be just perfect. Anyway, I will let you know, if I think it's not 😛.

**Comment by alyssaxuu** [comment]
Looking into this, I'm not sure what would actually work. There are actually cases where the user wants to move the toolbar on top of the camera (and it must be in this stacking order to keep the controls always in screen). And the snappy movement doesn't quite work for the camera either, I've looked into it. I will close it for now until I find a better way to go about it.

**Comment by ceskyDJ** [comment]
Hmm, I see. What about just reuse the system for the toolbar and don't take care of positioning between the camera and the toolbar, at least for now? I didn't know the importance of the possibility of overlapping the toolbar over the camera. If it's really useful in some cases, it probably should stay there. My biggest intention for this issue was the behavior of the camera circle near the ends of the screen. I need a stable interface that supports the subject of my recording. I think Screenity has the best UX from this point of view (I tried so many alternatives, when I wanted to leave Loom some time ago), but this is a thing I know I'll have troubles with. I usually make videos with SW bugs explanations (for free, so I don't want to stay recording for ages) and for syncs (mostly SW feature plans, making sure I know what and how it should do). It's very important for me to have stable recording SW, so I don't need to repeat the recording many times. I don't have notes most of the time, so every other recording could be lower quality (I forgot something, make a mistake, etc.). So this is why I created this issue. I think this is a pretty common requirement, but alternative SWs don't have such good UX, so it could help Screenity a lot.
